XmlValidator/spec_exercise.xml

```diff
@@ -297,4 +298,25 @@
     <notext/>
   </entry>
 
+  <entry>
+    <xpath>//entry/response/query//radio</xpath>
```

Comment:
The way this is setup currently means that EdTech can now use any number of radio tags in a query tag and then any number of button tags in a radio tag. I am not too sure that allowing any number of radio tags in a query tag is the desired behaviour. The structure you are aiming for should perhaps be (unless the spec clearly indicates otherwise):

`<query>`
 ` <radio>`
`    <button/>`
`    <button/>`
`    <button/>`
`    ...`
`  </radio>`
`</query>`